src/event.rs

```diff
@@ -126,6 +127,12 @@ impl<'a, N: Notify + 'a> input::ActionContext for ActionContext<'a, N> {
     fn last_modifiers(&mut self) -> &mut ModifiersState {
         &mut self.last_modifiers
     }
+
+    fn lines_to_scroll(&mut self, lines: f32) -> i32 {
```

Comment:
It might make more sense to put this method on the input processor so the logic can be reused with different `ActionContext`.